airflow/providers/amazon/aws/operators/emr_create_job_flow.py

```diff
@@ -70,7 +71,7 @@ def execute(self, context):
         if isinstance(self.job_flow_overrides, str):
             self.job_flow_overrides = ast.literal_eval(self.job_flow_overrides)
 
-        response = emr.create_job_flow(self.job_flow_overrides)
+        response = emr.create_job_flow(self.job_flow_overrides)     # type: ignore
```

Comment:
@mik-laj Thanks for review, I missed it.
This line is assigned by the ast module, is there any way to tell mypy about it?
I just annotated ignore.